### PR TITLE
Rename and docs update rebase

### PR DESCRIPTION
**Pull request by ayenpure**
This is a duplicate of MR #83 
It contains the changes for renaming `pan3D-viewer` to `geotrame`  and updating documentation wherever necessary
I created the MR from a new branch so that we do not lose Anne's branch in case I messed up the rebase.

### CODE REVIEW

**Review by johnkit — APPROVED** [review]
I know Abhi walked through everything, so let's go ahead and merge.

**Comment by ayenpure** [comment]
@annehaley Can you take a look once more and approve if this looks okay?

File: pyproject.toml
Comment (annehaley) [inline pyproject.toml:66]: I think this change is outdated, so you can take this out. I believe @johnkit had mentioned that the recursive dependency definition doesn't work in some cases, so we should use the explicit list.

**Pull request merged**
